src/Analysis/Engine/Test/CompletionTests.cs

```diff
@@ -1003,6 +1003,15 @@ class Simple(unittest.TestCase):
             }
         }
 
+        [TestMethod, Priority(0)]
+        public async Task WithWhitespaceAroundDot() {
+            using (var s = await CreateServerAsync()) {
+                var u = await s.OpenDefaultDocumentAndGetUriAsync("import sys\nsys  .  version\n");
```

Comment:
Do we care about `\r\n`?

Reply:
Not here, I believe there are parser tests for the line endings